llvm/test/MC/COFF/dwarf5lineinfo.s

```diff
@@ -0,0 +1,17 @@
+// RUN: llvm-mc -filetype=obj -triple x86_64-pc-windows-gnu %s -o - | llvm-readobj -r  - | FileCheck %s
+
+// CHECK: Relocations [
+// CHECK:  Section (4) .debug_line {
+// CHECK:    0x22 IMAGE_REL_AMD64_SECREL .debug_line_str (8)
+// CHECK:    0x2C IMAGE_REL_AMD64_SECREL .debug_line_str (8)
+// CHECK:    0x36 IMAGE_REL_AMD64_ADDR64 .text (0)
+// CHECK:  }
+
+main:                                
+	.file	0 "/" "test.c" 
+	.loc	0 1 0                           # test.c:1:0
+	retq
+.Linfo_string:
+.Linfo_string1:
+	.secrel32	.Linfo_string1
```

Comment:
Do you need these 3 lines? (They don't look like they contribute to the relocations, but I don't know how COFF works.)

Reply:
You're right, I tried around a bit and I can actually remove a lot more lines, will push the changes now